openfe/tests/setup/atom_mapping/conftest.py

```diff
@@ -41,3 +43,20 @@ def gufe_atom_mapping_matrix(lomap_basic_test_files_dir,
             molA_to_molB=val)
 
     return ligand_atom_mappings
+
+
+@pytest.fixture()
+def mol_pair_to_shock_perses_mapper() -> Tuple[SmallMoleculeComponent,
+                                               SmallMoleculeComponent]:
+    """
+        This pair of Molecules leads to an empty Atom mapping in
+        Perses Mapper with certain settings.
```

Comment:
```suggestion
    """
    This pair of Molecules leads to an empty Atom mapping in
    Perses Mapper with certain settings.
```

The indentation doesn't match the visual style.